### DISCUSSION TITLE
Add the possibility of translation

### DISCUSSION THREAD

**Discussion by Angel-bzh** [P3pp3rF1y/SophisticatedBackpacks#849] [General]
Good morning,

I come to ask if it is possible to give the possibility of a translation?
Thanks to crowdin I have translated the whole of Minecolonie into French and I would like to do the same for your mods.

Sincerely.

**Comment by P3pp3rF1y** [comment]
of course translation is possible, just need the french language file with the translation of what's in en_us.json
